eslint.config.js

```diff
@@ -53,8 +53,8 @@ export default [
 			'react/prop-types': 'off',
 			'react/no-unknown-property': ['error', { ignore: ['tw'] }],
 			// Disabled for PRs to be merged
-			'react/jsx-key': 'off',
-			'@typescript-eslint/no-unused-vars': 'off',
+			// 'react/jsx-key': 'off',
```

Comment:
why is this commented out?

Reply:
this PR removes the 'off' modifier, enabling both of these rules